qml/components/PhotoTextsListItem.qml

```diff
@@ -10,6 +12,7 @@ ListItem {
     property alias tertiaryText: tertiaryText //usually last message date
 
     property int unreadCount
+    property bool isSecret: false
```

Comment:
`false` is a default for `bool`, just like zero for `int`

Reply:
I actually like to get this type of overview - as you mentioned below, I try to make it consistent that way. ;)